crates/wasmi/src/engine/tests/regmach/op/unary/conversion.rs

```diff
@@ -310,11 +330,13 @@ macro_rules! iN_reinterpret_fN_tests {
                 const OP: &str = $op;
 
                 #[test]
+#[cfg_attr(miri, ignore)]
```

Comment:
Thanks a lot for your help! I feel it is better for me to apply this automated refactoring on my own since it is less work to apply myself than to review a huge external PR diff. I hope you can understand that.

Reply:
Oops, I missed those macros and that `cargo fmt` doesn't work inside them.